src/sage/geometry/polyhedron/combinatorial_polyhedron/face_iterator.pxd

```diff
@@ -5,10 +5,16 @@ from .face_data_structure       cimport face_t
 from .face_list_data_structure  cimport face_list_t
 from .combinatorial_face        cimport CombinatorialFace
 
+cdef enum FaceStatus:
+    not_initialized
+    initialized
+    ignore_subsets
+    only_visit_subsets
+
```

Comment:
I think the values should be all-caps, like `INITIALIZED`

Reply:
Sure. I was also thinking back and forth. I do Kotlin a lot nowadays, and there I would have chosen all-caps, but wasn't too sure, what looks better in the python world.

Done in 8c2f0f82fda8c5ac6fe4cc9db9d4256c16f24de5